cg/store/models.py

```diff
@@ -1075,6 +1075,33 @@ class IlluminaSequencingRun(InstrumentRun):
     __mapper_args__ = {"polymorphic_identity": DeviceType.ILLUMINA}
 
 
+class PacBioSequencingRun(InstrumentRun):
+    __tablename__ = "pacbio_sequencing_run"
+
+    id: Mapped[int] = mapped_column(ForeignKey("instrument_run.id"), primary_key=True)
+    well: Mapped[Str32]
+    plate: Mapped[int]
+    movie_time_hours: Mapped[int]
+    hifi_reads: Mapped[BigInt]
+    hifi_yield: Mapped[BigInt]
+    hifi_mean_read_length: Mapped[BigInt]
+    hifi_median_read_quality: Mapped[Str32]
+    percent_reads_passing_q30: Mapped[Num_6_2]
+    p0_percent: Mapped[Num_6_2]
+    p1_percent: Mapped[Num_6_2]
+    p2_percent: Mapped[Num_6_2]
+    polymerase_mean_read_length: Mapped[BigInt]
+    polymerase_read_length_n50: Mapped[BigInt]
+    polymerase_mean_longest_subread: Mapped[BigInt]
+    polymerase_longest_subread_n50: Mapped[BigInt]
+    control_reads: Mapped[BigInt]
+    control_mean_read_length: Mapped[BigInt]
+    control_mean_read_concordance: Mapped[Num_6_2]
+    control_mode_read_concordance: Mapped[Num_6_2]
```

Comment:
any aliases or defaults for this that you could think of? @ChrOertlin @Vince-janv